docusaurus/docs/get-started/get-started.mdx

```diff
@@ -169,53 +165,107 @@ The file structure should look like this:
 
-### `mage -v`
+Run the following commands to get started:
 
-Builds backend plugin binaries for Linux, Windows and Darwin.
+    * cd ./orgName-pluginName-app
+    * npm install to install frontend dependencies.
+    * npm exec playwright install chromium to install e2e test dependencies.
+    * npm run dev to build (and watch) the plugin frontend code.
+    * mage -v build:backend to build the plugin backend code. Rerun this command every time you edit your backend files.
+    * docker-compose up to start a grafana development server.
+    * Open http://localhost:3000 in your browser to create a dashboard to begin developing your plugin.
 
-### `npm run build`
+Note: We strongly recommend creating a new Git repository by running git init in ./org-pluginname-app before continuing.
 
-Compiles and bundles the project using Webpack in production mode.
+    * Learn more about Grafana Plugin Development at https://grafana.com/developers/plugin-tools
 
-### `npm run dev`
+```
 
-Runs Webpack in watch mode for development, continually monitoring for changes.
+### Install dependencies
 
-### `npm run typecheck`
+<CodeSnippets
+  snippets={[
+    { component: InstallNPM, label: 'npm' },
+    { component: InstallPNPM, label: 'pnpm' },
+    { component: InstallYarn, label: 'yarn' },
+  ]}
+  groupId="package-manager"
+  queryString="current-package-manager"
+/>
 
-Performs a type-checking process on the frontend code using TypeScript.
+### Build the frontend
 
-### `npm run server`
+To build the plugin in watch mode for development, continually monitoring for changes, run:
+<CodeSnippets
+  snippets={[
+    { component: BuildFEDevNPM, label: 'npm' },
+    { component: BuildFEDevPNPM, label: 'pnpm' },
+    { component: BuildFEDevYarn, label: 'yarn' },
+  ]}
+  groupId="package-manager"
+  queryString="current-package-manager"
+/>
 
-Launches the [Grafana development server](https://grafana.com/developers/plugin-tools/get-started/set-up-development-environment) using Docker.
 
-### `npm run sign`
+To build for production, run:
 
-Signs the Grafana plugin using the latest version of `@grafana/sign-plugin`.
+<CodeSnippets
+  snippets={[
+    { component: BuildFEProdNPM, label: 'npm' },
+    { component: BuildFEProdPNPM, label: 'pnpm' },
+    { component: BuildFEProdYarn, label: 'yarn' },
+  ]}
+  groupId="package-manager"
+  queryString="current-package-manager"
+/>
 
-:::info
+### Build the backend
 
-For a complete list of available CLI commands, refer to the [CLI reference](/reference/cli-commands.md).
+If your plugin includes a [backend](../key-concepts/backend-plugins/index.md) component, you can build using mage:
 
-:::
+```shell
+mage -v build:linux
+```
+
+#### Build targets
+
+| Option         | Description                                  | Example               |
+| -------------- | -------------------------------------------- | --------------------- |
+| `build:[arch]` | Builds a binary for a specific architecture. | `mage -v build:Linux` |
+
```

Comment:
There is a mismatch between `amd` and `arm64` and the docker container which we should likely put a note about here. Previously we forced amd in the docker platform but that causes the backend delve dev env to fail to run on apple silicon.

Also not really sure if this should be added here but we have more than just `build:[arch]` as available targets. e.g.

```
| `build:backend` | build a production build for the current platform | `mage -v build:backend` |
```

Maybe we circle back to the whole backend build commands and expand them later on.

Reply:
Given this is primarily a re-org is it ok to address this later?